src/refract/Element.cc

```diff
@@ -217,56 +227,56 @@ namespace refract
                     MapKeyToMember keysBase;
                     MapNameToRef refBase;
 
-                    for (RefractElements::iterator it = value.begin(); it != value.end(); ++it) {
-                        if (MemberElement* member = TypeQueryVisitor::as<MemberElement>(*it)) {
+                    for (auto& it : value) {
```

Comment:
Add a C assert on the pointer element. I imagine a `nullptr` here hard to debug as the below line would seem like it couldn't interpret something as a `MemberElement`.